text/0013-delete-enqueued-updates.md

```diff
@@ -0,0 +1,142 @@
+- Title: Abort enqueued updates
+- Start Date:
+- Specification PR: https://github.com/meilisearch/specifications/pull/13
+- MeiliSearch Issue: 
+    - https://github.com/meilisearch/MeiliSearch/issues/1104
+- Parent Spec: https://github.com/meilisearch/specifications/pull/12
+
+
+# Abort enqueued updates
+
+## 1. Feature Description and Interaction
+
+### Summary
+
+Allow to abort messages in the update queue or to empty the entire queue. An aborted update will have the status `aborted`.
+
+Questions not answered:
+- We have an attribute on update info `processedAt`. What becomes this attribute if we add the possibility to abort an update? We already use this term in the case of a failed update.
```

Comment:
I would say that this becomes that date time at which its status changed from pending to aborted.

Reply:
So should we keep `processedAt` or should we change to `abortedAt` in the case of an abortion?